projects/discovery/pkg/fds/updater.go

```diff
@@ -115,7 +114,7 @@ func (u *Updater) UpstreamUpdated(upstream *v1.Upstream) {
 
 func (u *Updater) UpstreamAdded(upstream *v1.Upstream) {
 	// upstream already tracked. ignore.
-	key := translator.UpstreamToClusterName(upstream.GetMetadata().Ref())
+	key := upstream.GetMetadata().Ref().Key()
```

Comment:
this was part of the cleanup i carried over from https://github.com/solo-io/gloo/pull/10403. in some plugins (etc) we are just using the "cluster name" as key for a map used internally, and i changed those instances to use Ref.Key instead